### PR TITLE
chore: bundle flagd new proto schems

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
## This PR

Adds support to bundle new grpc proto schemas of flagd. This is one of several steps to push flagd to use new schemas released with https://github.com/open-feature/flagd-schemas/releases/tag/protobuf-v0.5.3

### CODE REVIEW

File: providers/flagd/pom.xml [hunk 1]
```diff
@@ -199,6 +199,36 @@
                             </arguments>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>copy-new-schema-definition</id>
```
Comment (toddbaert) [inline providers/flagd/pom.xml]:
```suggestion
                        <id>copy-evaluation.proto</id>
```

File: providers/flagd/pom.xml [hunk 2]
```diff
@@ -199,6 +199,36 @@
...
+                            <!-- run: cp schemas/protobuf/flagd/evaluation/v1/evaluation.proto src/main/proto/ -->
+                            <executable>cp</executable>
+                            <arguments>
+                                <argument>schemas/protobuf/flagd/evaluation/v1/evaluation.proto</argument>
+                                <argument>src/main/proto/</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>copy-new-sync-definition</id>
```
Comment (toddbaert) [inline providers/flagd/pom.xml]:
```suggestion
                        <id>copy-sync.proto</id>
```

File: providers/flagd/src/main/proto/.gitignore [hunk 3]
```diff
@@ -1,2 +1,4 @@
+evaluation.proto
 schema.proto
-sync_service.proto
\ No newline at end of file
+sync.proto
+sync_service.proto
```
Comment (toddbaert) [inline providers/flagd/src/main/proto/.gitignore]: Maybe just do `*.proto`

**Review by toddbaert — APPROVED** [review]

File: providers/flagd/src/main/proto/.gitignore (hunk 3 above)
Comment (Kavindu-Dodan) [reply providers/flagd/src/main/proto/.gitignore]: yeah, will update with spot bug exclusions

Comment (Kavindu-Dodan) [reply providers/flagd/src/main/proto/.gitignore]: done :)

**Pull request merged**